### PR TITLE
HWKAPM-681 Fix alerts publisher to supply the dataId needed by the al…

### PR DESCRIPTION
**Pull request by objectiser**
…erts engine. Also updated the env var names to be consistent with other names used by APM

### CODE REVIEW

**Comment by objectiser** [comment]
@pavolloffay Could you review this please?

Reviewer notes:
* Found issue when testing alerts integration - the 'dataId' field needs to be supplied as otherwise alerts are not triggered
* Have changed the env variable names used by the APM server to communicate with Alerts, to be more consistent with the APM variables
* Once this issue was resolved, discovered another issue hwkapm 684, which will need to be investigated later - so for now have commented out publishing fragment completion time events to the alerts engine - which are not important for this release.

File: api/src/main/java/org/hawkular/apm/api/utils/PropertyUtil.java [hunk 1]
```diff
@@ -34,6 +34,21 @@
...
+     */
+    public static final String HAWKULAR_URI = "HAWKULAR_URI";
+
+    /**
+     * Username for authentication against the Hawkular Server
+     */
+    public static final String HAWKULAR_USERNAME = "HAWKULAR_USERNAME";
+
+    /**
+     * Password for authentication against the Hawkular Server
```
Comment (pavolloffay) [inline api/src/main/java/org/hawkular/apm/api/utils/PropertyUtil.java]: Can we use the same name for hawkular server? Either `Hawkular Server` or `Hawkular Services` I prefer the second one.

**Comment by pavolloffay** [comment]
Results :
Failed tests: 
  EventTest.eventSetsCategory:85 expected:<[APM]> but was:<[eventSetsCategory]>
  EventTest.eventSetsDataSource:68 expected:<null> but was:<APM>

**Comment by objectiser** [comment]
@pavolloffay Woops - can't believe I hadn't run those tests locally - anyway should be fixed now.

**Comment by pavolloffay** [comment]
re-review LGTM :+1:

**Comment by objectiser** [comment]
@pavolloffay Thanks!

**Pull request merged**